doc/source/conf.py

```diff
@@ -23,6 +25,43 @@
 except TypeError:
     pass
 
+from sphinx.application import Sphinx
+
+HERE = Path(__file__).parent
+
+
+def convert_md_to_ipynb(app: Sphinx, *args, **kwargs):
+    import subprocess
+    import sys
+    print("Converting Markdown files to IPyNB...")
+    subprocess.check_call(
+        [
+            sys.executable,
+            "-m",
+            "jupytext",
+            "--to",
+            "ipynb",
+            f"{HERE / 'regression' / '*.md'}",
+        ]
+    )
+
+
+def cleanup_ipynb(app: Sphinx, *args, **kwargs):
+    import nbformat
+
+    for path in (HERE / "regression").glob("*.ipynb"):
+        with open(path) as f:
+            nb = nbformat.read(f, as_version=4)
+            nb.cells = [cell for cell in nb.cells if "true" not in cell.metadata.get("ignore-when-converting", [])]
+
+        with open(path, "w") as f:
+            nbformat.write(nb, f)
+            print(f"Cleaned up {path}")
+
+
+def setup(app):
+    app.connect("builder-inited", convert_md_to_ipynb)
+    app.connect("builder-inited", cleanup_ipynb)
```

Comment:
With these extensions, at least the downloadable IPyNB is cleaner, if not the Markdown-based notebook.